Makefile

```diff
@@ -564,6 +564,12 @@ awx-kube-dev-build: Dockerfile.kube-dev
 	    --cache-from=$(DEV_DOCKER_TAG_BASE)/awx_kube_devel:$(COMPOSE_TAG) \
 	    -t $(DEV_DOCKER_TAG_BASE)/awx_kube_devel:$(COMPOSE_TAG) .
 
+awx-kube-build: Dockerfile
```

Comment:
added this so that we can build test version of the kube build to deploy on cloud